dash/development/_py_components_generation.py

```diff
@@ -95,10 +96,10 @@ def __init__(self, {default_argtext}):
     if "children" in props:
```

Comment:
I think it would be better to change it like this.

```suggestion
    if "children" in list_of_valid_keys:
```

Reply:
Thanks! I applied it by adding you as co-author.